src/AsmResolver/AsmResolver.csproj

```diff
@@ -7,7 +7,7 @@
         <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
         <NoWarn>1701;1702;NU5105</NoWarn>
         <Nullable>enable</Nullable>
-        <TargetFrameworks>net6.0;netcoreapp3.1;netstandard2.0</TargetFrameworks>
+        <TargetFrameworks>net6.0;netcoreapp3.1;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
Should we add netstandard2.1 targets to the other csproj as well, for consistency?

Reply:
I considered it, though since they aren't actually making use of these APIs yet I wasn't sure if it would make sense to. Do you have a preference here?

Reply:
Let's add them to all packages, even if it is just for consistency sake.